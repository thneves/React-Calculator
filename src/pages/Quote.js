import React from 'react';
import '../styles/Quote.css';

const Quote = () => (
  <div className="quote-div">
    <h3 className="quote">&quot;Without mathematics, thereâ€™s nothing you can do. Everything around you is mathematics. Everything around you is numbers.&quot;</h3>
    <p className="author">- Shakuntala Devi</p>
  </div>
);

export default Quote;
