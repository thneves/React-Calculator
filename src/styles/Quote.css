.quote-div {
  background-color: #a7bdbe;
  height: 100vh;
  display: flex;
  flex-direction: column;
  align-items: center;
  gap: 20px;
}

.quote {
  color: #042324;
  width: 900px;
  padding-top: 100px;
  font-family: Verdana, Geneva, Tahoma, sans-serif;
  font-style: italic;
  font-size: 20px;
}

.author {
  font-family: Impact, Haettenschweiler, 'Arial Narrow Bold', sans-serif;
  font-weight: 500;
  font-size: 30px;
  color: #083d3f;
}
