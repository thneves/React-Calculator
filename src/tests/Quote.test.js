import React from 'react';
import TestRenderer from 'react-test-renderer';
import Quote from '../pages/Quote';

it('should render the Quote page component', () => {
  const quote = TestRenderer
    .create(<Quote />)
    .toJSON();
  expect(quote).toMatchSnapshot();
});
