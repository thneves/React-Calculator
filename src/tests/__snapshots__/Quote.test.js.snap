// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the Quote page component 1`] = `
<div
  className="quote-div"
>
  <h3
    className="quote"
  >
    "Without mathematics, thereâ€™s nothing you can do. Everything around you is mathematics. Everything around you is numbers."
  </h3>
  <p
    className="author"
  >
    - Shakuntala Devi
  </p>
</div>
`;
